rpcs3/Emu/Cell/Modules/cellPngDec.cpp

```diff
@@ -724,7 +724,7 @@ s32 pngDecodeData(ppu_thread& ppu, PHandle handle, PStream stream, vm::ptr<u8> d
 
 		stream->cbDispParam->nextOutputImage = disp_param->nextOutputImage;
 
-		streamInfo->decodedStrmSize = stream->buffer->cursor;
+		streamInfo->decodedStrmSize = static_cast<unsigned int>(stream->buffer->cursor);
```

Comment:
why the sudden use of unsigned int instead of u32?

Reply:
Misclicked on something I guess while trying to figure out the type.